src/js/variant-widget/filters/eva-consequence-type-filter-form-panel.js

```diff
@@ -208,7 +208,7 @@ EvaConsequenceTypeFilterFormPanel.prototype = {
             model: 'Tree Model',
             proxy: {
                 type: 'memory',
-                data: this.consequenceTypes,
+                data:_.sortBy(this.consequenceTypes, 'name'),
```

Comment:
This doesn't seem to be working, given that the entries don't appear sorted? Or does it work only for the first level of the tree?